### PR TITLE
Azure DPS documentation - final draft for revision

### PR DESCRIPTION
**Pull request by abaravikova666**
Here's the final draft of the Azure DPS documentation.
As always, your comments and suggestions are highly appreciated.

### CODE REVIEW

File: docs/Azure_IoT_Integration_Guide/Azure_IoT_DPS_integration/Configuring_Azure_IoT_DPS_integration_extension.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+    * **Enrollment group key:** paste the Primary Key copied from the Enrollment group.
+    * **Device Provisioning Service hostname** can be left as it is: `global.azure-devices-provisioning.net`
+
+![Enter ID Scope, Enrollment group key and click Save](images/dps-8.png "Enter ID Scope, Enrollment group key and click Save")
+
+## What happens next
+
+After you set up Azure DPS and configure your Azure DPS extension in Coiote DM, device provisioning and reprovisioning are automated. You aren’t required to make any changes in Coiote DM.
+
+Importing and exporting devices, upgrading device firmware, and setting observations — all these operations happen at the level of individual hubs. The detailed instructions are provided in our [Azure IoT Hub documentation](https://iotdevzone.avsystem.com/docs/Azure_IoT_Integration_Guide/Azure_IoT_Hub_integration/Device_operations/Overview/).
```
Comment (Passarinho4) [inline docs/Azure_IoT_Integration_Guide/Azure_IoT_DPS_integration/Configuring_Azure_IoT_DPS_integration_extension.md:80]: I'm not sure that this part is clear. I understand that, but I guess that may be messy for people who do not have enough understanding and knowledge about that. When the DPS extension is configured all devices are provisioned to the proper IoT Hub using that DPS service. Linking IoT hub documentation may suggest that the user is required to add IoT hub somehow. 

But this is just my reflection.

**Review by Passarinho4 — CHANGES_REQUESTED** [review]

File: docs/Azure_IoT_Integration_Guide/Azure_IoT_DPS_integration/Configuring_Azure_IoT_DPS_integration_extension.md [hunk 2]
```diff
@@ -0,0 +1,85 @@
+# Configure integration
+
+Azure IoT Hub Device Provisioning Service (DPS) is a helper service for Azure IoT Hub that allows registering large numbers of devices without any human intervention. Azure DPS ensures high availability by providing load balancing across multiple hubs and supporting reprovisioning based on a change in the device. For more information about Azure DPS, refer to the [official documentation](https://docs.microsoft.com/en-us/azure/iot-dps/about-iot-dps).
```
[the remark this answers is not in the document]
Comment (abaravikova666) [reply docs/Azure_IoT_Integration_Guide/Azure_IoT_DPS_integration/Configuring_Azure_IoT_DPS_integration_extension.md:3]: slightly changed: "... that enables zero-touch provisioning of IoT devices at scale". I guess that "at scale" is also a sort of a buzzword :)

File: docs/Azure_IoT_Integration_Guide/Azure_IoT_DPS_integration/Configuring_Azure_IoT_DPS_integration_extension.md [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+
+To enable communication and data flow between Azure DPS and the Coiote DM platform, you first need to integrate them using the dedicated extension module in Coiote DM. Follow the instruction below to learn how to do it.
```
[the remark this answers is not in the document]
Comment (abaravikova666) [reply docs/Azure_IoT_Integration_Guide/Azure_IoT_DPS_integration/Configuring_Azure_IoT_DPS_integration_extension.md:5]: good point. I've elaborated a bit based on your comment

File: docs/Azure_IoT_Integration_Guide/Azure_IoT_DPS_integration/Configuring_Azure_IoT_DPS_integration_extension.md [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+
+
+## Prerequisites
+
+* An active DPS account [with linked IoT hubs](https://docs.microsoft.com/en-us/azure/iot-dps/quick-setup-auto-provision#create-a-new-iot-hub-device-provisioning-service).
```
[the remark this answers is not in the document]
Comment (abaravikova666) [reply docs/Azure_IoT_Integration_Guide/Azure_IoT_DPS_integration/Configuring_Azure_IoT_DPS_integration_extension.md:10]: went for the first one (removed `account`).

File: docs/Azure_IoT_Integration_Guide/Azure_IoT_DPS_integration/Configuring_Azure_IoT_DPS_integration_extension.md [hunk 5]
```diff
@@ -0,0 +1,85 @@
...
+* An active IoT Hub with hub owner access permissions. [Read here](https://docs.microsoft.com/en-us/azure/iot-hub/iot-hub-create-through-portal) how to create a hub.
+* A Coiote DM user account with permissions to use the integration extension.
+
+
+## Create an Enrollment group
+
+This step describes how to create an Enrollment group. If you already have one, proceed to the [Get the credentials](#get-the-credentials) section.
+
+A device can be enrolled in Azure DPS either [individually or as a group of devices](https://docs.microsoft.com/en-us/azure/iot-dps/tutorial-provision-device-to-hub#enroll-the-device) that share a specific attestation mechanism. For the integration with Coiote DM, you need to choose the Enrollment group option.
```
[the remark this answers is not in the document]
Comment (abaravikova666) [reply docs/Azure_IoT_Integration_Guide/Azure_IoT_DPS_integration/Configuring_Azure_IoT_DPS_integration_extension.md:19]: good point, added a brief explanation that Coiote doesn't support indivial enrollments at this moment

File: docs/Azure_IoT_Integration_Guide/Azure_IoT_DPS_integration/Configuring_Azure_IoT_DPS_integration_extension.md (hunk 1 above)
Comment (abaravikova666) [reply docs/Azure_IoT_Integration_Guide/Azure_IoT_DPS_integration/Configuring_Azure_IoT_DPS_integration_extension.md:80]: I'm not sure now how to put it in another way, but will think about it.

File: docs/Azure_IoT_Integration_Guide/Azure_IoT_DPS_integration/Configuring_Azure_IoT_DPS_integration_extension.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (abaravikova666) [reply docs/Azure_IoT_Integration_Guide/Azure_IoT_DPS_integration/Configuring_Azure_IoT_DPS_integration_extension.md]: link to the changes: https://github.com/AVSystem/developer-zone/pull/34/commits/448205dfa993bb875785439f79b6140ef0d9e9fb

File: docs/Azure_IoT_Integration_Guide/Azure_IoT_DPS_integration/Configuring_Azure_IoT_DPS_integration_extension.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (abaravikova666) [reply docs/Azure_IoT_Integration_Guide/Azure_IoT_DPS_integration/Configuring_Azure_IoT_DPS_integration_extension.md]: https://github.com/AVSystem/developer-zone/pull/34/commits/448205dfa993bb875785439f79b6140ef0d9e9fb

File: docs/Azure_IoT_Integration_Guide/Azure_IoT_DPS_integration/Configuring_Azure_IoT_DPS_integration_extension.md [hunk 6]
```diff
@@ -0,0 +1,85 @@
...
+    ![Click Add enrollment group in the Manage enrollments panel](images/dps-1.png "Click Add enrollment group in the Manage enrollments panel")
+
+4. Provide the group name and select the **Symmetric key** option from the Attestation Type. Make sure that the **Auto-generate keys** checkbox is ticked. Click **Save**.
+
+![Provide the group name and select the Symmetric key option](images/dps-2.png "Provide the group name and select the Symmetric key option")
+
+You’ve just created your enrollment group. The generated Symmetric Key will be used in the next step.
+
+!!! info
+    On the Enrollment Group Details page you can also select to which hub the enrollment group will be assigned. The list of hubs that are connected to this DPS is displayed in the drop-down menu under **Select the IoT hubs this group can be assigned to**. To add more hubs to this list, click **Link a new IoT hub** or go to the Linked IoT hubs page from the left pane.
```
[the remark this answers is not in the document]
Comment (abaravikova666) [reply docs/Azure_IoT_Integration_Guide/Azure_IoT_DPS_integration/Configuring_Azure_IoT_DPS_integration_extension.md:36]: @wodomierz  in your opinion, what's the most important thing about high availability that is worth mentioning here?

**Pull request merged**